core/controlplane/config/config.go

```diff
@@ -116,22 +116,22 @@ func NewDefaultCluster() *Cluster {
 			ManageCertificates:          true,
 			HyperkubeImage:              model.Image{Repo: "quay.io/coreos/hyperkube", Tag: k8sVer, RktPullDocker: false},
 			AWSCliImage:                 model.Image{Repo: "quay.io/coreos/awscli", Tag: "master", RktPullDocker: false},
-			CalicoNodeImage:             model.Image{Repo: "quay.io/calico/node", Tag: "v1.1.3", RktPullDocker: false},
-			CalicoCniImage:              model.Image{Repo: "quay.io/calico/cni", Tag: "v1.7.0", RktPullDocker: false},
-			CalicoPolicyControllerImage: model.Image{Repo: "quay.io/calico/kube-policy-controller", Tag: "v0.5.4", RktPullDocker: false},
-			CalicoCtlImage:              model.Image{Repo: "quay.io/calico/ctl", Tag: "v1.1.3", RktPullDocker: false},
-			ClusterAutoscalerImage:      model.Image{Repo: "gcr.io/google_containers/cluster-proportional-autoscaler-amd64", Tag: "1.0.0", RktPullDocker: false},
+			CalicoNodeImage:             model.Image{Repo: "quay.io/calico/node", Tag: "v1.2.0", RktPullDocker: false},
+			CalicoCniImage:              model.Image{Repo: "quay.io/calico/cni", Tag: "v1.8.3", RktPullDocker: false},
+			CalicoPolicyControllerImage: model.Image{Repo: "quay.io/calico/kube-policy-controller", Tag: "v0.6.0", RktPullDocker: false},
```

Comment:
[v0.6.0](https://github.com/projectcalico/k8s-policy/releases/tag/v0.6.0) does seem to have important fixes but it isn't included in [the latest release of calico v2.1.5](http://docs.projectcalico.org/v2.1/releases/) as a whole.
I'm personally ok with following latest releases of calico per component basis like you did but is it your intention?
Which would you like, following each release per component, or following only the whole release?